src/Uno.UI/UI/Xaml/ResourceDictionaryExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿#nullable enable
+
+using System.ComponentModel;
+using Windows.UI.Xaml;
+
+namespace Uno.UI.Xaml;
+
+/// <summary>
+/// Helper for XAML code generation. Not intended to be used in apps outside of XAML generator.
+/// </summary>
+[EditorBrowsable(EditorBrowsableState.Never)]
+public static class ResourceDictionaryExtensions
+{
+	public static ResourceDictionary AddMergedDictionaries(this ResourceDictionary dictionary, params ResourceDictionary[] mergedDictionaries)
```

Comment:
I wonder if those two should be placed in:
https://github.com/unoplatform/uno/blob/57d0fed00d8215014544557844e272bf2c375863/src/Uno.UI/Helpers/MarkupHelper.cs#L20

Reply:
I'm fine either way. I'll move them.